sdx_lc/swagger/swagger.yaml

```diff
@@ -642,6 +644,106 @@ paths:
       x-openapi-router-controller: sdx_lc.controllers.user_controller
 components:
   schemas:
+    connection_v2:
+      required:
+      - description
+      - endpoints
+      - id
+      - name
+      - notifications
+      - qos_metrics
+      - scheduling
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        endpoints:
+          type: array
+          items:
+            $ref: '#/components/schemas/port'
```

Comment:
The array of endpoints are of the below form, so `'#/components/schemas/port'` will not work:

```json
  "endpoints": [
    {
      "port_id": "urn:sdx:port:ampath.net:Ampath3:50",
      "vlan": "777"
    },
    {
      "port_id": "urn:sdx:port:sax.net:Sax01:41",
      "vlan": "55:90"
    },
    {
      "port_id": "urn:sdx:port:ampath.net:Ampath3:50",
      "vlan": "untagged"
    }
  ],
```

What I have found when working on https://github.com/atlanticwave-sdx/sdx-controller/pull/297 is that this will work:

```yaml
        endpoints:
          type: array
          items:
            type: object
            properties:
              port_id:
                type: string
              vlan:
                type: string
```

Reply:
right, updated.